static/src/javascripts/projects/common/modules/atoms/youtube.spec.js

```diff
@@ -0,0 +1,30 @@
+// @flow
+import { checkElemsForVideos } from 'common/modules/atoms/youtube';
+
+jest.mock('common/modules/atoms/youtube-player', () => ({
+    initYoutubePlayer: jest.fn(() => null),
+}));
+
+jest.mock('common/modules/atoms/youtube-tracking', () => ({
+    initYoutubeEvents: jest.fn(() => null),
+    trackYoutubeEvents: jest.fn(() => null),
+}));
+
+describe('youtube', () => {
+    it('adds player to page', () => {
+        if (document.body) {
+            document.body.innerHTML =
+                '<div data-media-atom-id="atomId" class="u-responsive-ratio u-responsive-ratio--hd youtube-media-atom">' +
+                '<div id="youtube-assetId"></div>' +
+                '</div>';
```

Comment:
Could probably get some value using [template literals](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals) here

Reply:
Yes that's a good idea.  I'll do it!